### DISCUSSION TITLE
SSH is working fine on any console, cyberduck is not connecting to a server.

### DISCUSSION THREAD

**Discussion by Doinkey** [iterate-ch/cyberduck#14727] [Q&A]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**

When I log into a server through terminal.app, everything works fine. But when I put in the exact same information on an ssh connection through cyberduck, it refuses to connect. I have also tried on multiple devices which all produce the same outcome. When I check the log file on the server, it doesnt even show that I tried to connect when I tried through cyberduck. 

**To Reproduce**
try to connect to a server and it refuses

**Expected behavior**
I expected it to connect like it usually does.

**Screenshots**
<img width="587" alt="image" src="https://github.com/iterate-ch/cyberduck/assets/103324209/c28a018e-64d4-447f-abcc-a3a392345543">

**Desktop (please complete the following information):**
 - OS: macOS and Windows 10 & 11
 - Version 11.7.6

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.

[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/11593753/cyberduck.log)

**Additional context**

**Comment by tatze96** [comment]
Unfortunately, the attached log file only contains a few lines and doesn't include the mentioned error. Can you please verify that [debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level) is enabled, re-create the error, and re-attach the log file afterward?
